### PR TITLE
Adds guard for improved testing

### PR DESCRIPTION
**Pull request by Ricoch**
Adds guard for improved testing

### CODE REVIEW

**Comment by Alxzu** [comment]
Hey, I never heard about Guard gem before, not sure but I think there are two files repeated, right? Do you need both? Also not sure what are those watches over specs files, what are the purpose of those? It will be nice to have this gem with something like 
```
guard 'livereload' do
  watch(%r{app/.+\.(erb|haml)})
  watch(%r{app/helpers/.+\.rb})
  watch(%r{public/.+\.(css|js|html)})
  watch(%r{config/locales/.+\.yml})
end
```
to enable hot reload on the code

File: Gemfile [hunk 1]
```diff
@@ -30,6 +30,13 @@ gem 'sendgrid', '~> 1.2.4'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+
+# Use guard gem for improved testing
+gem 'guard', '~> 2.14'
+gem 'guard-rspec', '~> 4.7', '>= 4.7.3'
+gem 'rb-fsevent', '~> 0.10.3'
+
+
```
Comment (letiesperon) [inline Gemfile]: Delete this extra line

File: bin/guard [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```
Comment (letiesperon) [inline bin/guard]: Do you really need all these files? I've never used this gem

Comment (Ricoch) [reply bin/guard]: It seems both files within bin were unnecessary

**Review by letiesperon — APPROVED** [review]

**Comment by Ricoch** [comment]
@Alxzu Thanks, didn't realize both files were exactly the same. It also seems like both those files were unnecessary so I removed them both.

**Review by Alxzu — APPROVED** [review]

**Pull request merged**